README.md

```diff
@@ -647,10 +647,71 @@ type C = Omit<A, B>;
 
 However, Flow implementation is stricter in this case, as B have a property that A does not have, it would rise an error. In Typescript, however, they would be ignored. 
 
+## Private properties in classes
+
+### flow
+
+```js
+class SomeClass {
+  prop: string
+  #prop2: string
+  #prop3: string = "default value"
+
+  constructor(prop: string, prop2: string) {
+    this.prop = prop
+    this.#prop2 = prop2
+  }
+}
+```
+
+### TypeScript
```

Comment:
This example can be removed, or moved to the part below (above Object callable property), since it duplicates the part that starts with:

> Also, TypeScript have some syntactic sugar for setting the props through the constructor